metric-schema-java/src/main/java/com/palantir/metric/schema/UtilityGenerator.java

```diff
@@ -555,18 +562,26 @@ private static void generateMetricFactoryBuilder(
         } else {
             abstractBuildMethodBuilder.addAnnotation(CheckReturnValue.class);
         }
-        MethodSpec abstractBuildMethod = abstractBuildMethodBuilder.build();
-        MethodSpec abstractBuildMetricName = MethodSpec.methodBuilder("buildMetricName")
+        buildStageInterfaceBuilder.addMethod(abstractBuildMethodBuilder.build());
+
+        if (isHistogram) {
+            buildStageInterfaceBuilder.addMethod(MethodSpec.methodBuilder("build")
```

Comment:
pedantic: can we use a more descriptive method name and also generate some JavaDoc?

```suggestion
            buildStageInterfaceBuilder.addMethod(MethodSpec.methodBuilder("buildWithHistograms")
```

Reply:
Personally, I'd prefer if we named this `build` to be consistent with the `TaggedMetricRegistry` API.

I don't think the "WithHistograms" qualifier adds much value - this is obvious from the method signature.